src/main/java/uk/gov/pay/publicauth/resources/PublicAuthResource.java

```diff
@@ -189,13 +189,13 @@ private void validatePayloadHasFields(JsonNode payload, List<String> expectedFie
                 .filter(expectedKey -> !payload.has(expectedKey))
                 .collect(Collectors.toList());
         if (!missingFields.isEmpty()) {
-            throw new ValidationException("Missing fields: [" + Joiner.on(", ").join(missingFields) + "]");
+            throw new ValidationException(format("Missing fields: [%s]", String.join(", ", missingFields)));
```

Comment:
Changing this to `throw new ValidationException("Missing fields: " + missingFields);` would have the same effect because of how `List`s `toString()`.

Reply:
If you say so, I believe you. Amended